test/model/binding.dart

```diff
@@ -555,10 +555,13 @@ class FakeAndroidNotificationHostApi implements AndroidNotificationHostApi {
   }
   List<AndroidNotificationHostApiNotifyCall> _notifyCalls = [];
 
+  Iterable<StatusBarNotification> get activeNotifications => _activeNotifications.values;
+  final Map<(int, String?), StatusBarNotification> _activeNotifications = {};
+
   final Map<String, MessagingStyle?> _activeNotificationsMessagingStyle = {};
 
-  /// Clears all active notifications that have been created via [notify].
-  void clearActiveNotifications() {
+  /// Clears all [MessagingStyle] of active notifications that have been created via [notify].
+  void clearActiveNotificationsMessagingStyle() {
     _activeNotificationsMessagingStyle.clear();
   }
```

Comment:
Why clear only `_activeNotificationsMessagingStyle` and not `_activeNotifications`? It seems like that'll leave the two inconsistent with each other in a way that isn't realistic for how the real Android system behaves.

So I think the existing (in main) name and doc for this method are good, and it just needs to clear the new data structure as well as the old.